tests/unit_tests/PreprocessorTests.cpp

```diff
@@ -194,7 +194,7 @@ TEST_CASE("Function macro (keyword as formal argument)", "[preprocessor]") {
 }
 
 TEST_CASE("Macro pasting (identifiers)", "[preprocessor]") {
-    auto& text = "`define FOO(x,y) x   ``   _blah``y\n`FOO(   bar,    _BAZ)";
+    auto& text = "`define FOO(x,y) x``_blah``y\n`FOO(   bar,    _BAZ)";
```

Comment:
In most prior conversation I think we had decided that we wanted to maintain the macro-paste-ignoring-whitespace feature, what is the reason for getting rid of it? does HRT code actually use the spacing around a macro paste operator?